fenix/app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -513,9 +449,10 @@ class BookmarksTest {
         }
     }
 
+    // TestRail link: https://testrail.stage.mozaws.net/index.php?/cases/view/325644
     @SmokeTest
     @Test
-    fun deleteMultipleSelectionTest() {
+    fun deleteMultipleSelectedBookmarksTest() {
```

Comment:
Let's keep just this and openMultipleSelectedBookmarksInANewTabTest as smoke tests, you can remove the tag from openMultipleSelectedBookmarksInPrivateTabTest and the other multiple selection tests.

Reply:
Done !